waspc/src/Parser/EntityPSL.hs

```diff
@@ -0,0 +1,19 @@
+module Parser.EntityPSL
+    ( entityPSL
+    ) where
+
+import Text.Parsec.String (Parser)
+
+import qualified Wasp.EntityPSL
+import qualified Parser.Common as P
+import qualified Lexer as L
+
```

Comment:
Maybe you could explain here what PSL stands for?

Reply:
right, good idea! Added explanation - it stands for Prisma Schema Language